wolfcrypt/src/port/nxp/ksdk_port.c

```diff
@@ -524,6 +538,79 @@ int mp_exptmod(mp_int *G, mp_int *X, mp_int *P, mp_int *Y)
     return res;
 }
 
+int mp_exptmod_nct (mp_int * G, mp_int * X, mp_int * P, mp_int * Y)
+{
+    /* use hardware implementation even for non-constant time operations */
+    return mp_exptmod(G, X, P, Y);
+}
+
+#if !defined(NO_DH) || !defined(NO_DSA) || !defined(NO_RSA) || \
+    defined(WOLFSSL_KEY_GEN)
+int mp_prime_is_prime_ex(mp_int* a, int t, int* result, WC_RNG* rng)
+{
+    int res = MP_OKAY;
+    int szA;
+
+    szA = mp_unsigned_bin_size(a);
+    if (szA <= LTC_MAX_INT_BYTES) {
+        uint16_t sizeA, sizeB;
+        uint8_t *ptrA, *ptrB;
```

Comment:
Should initialize ptrA and ptrB to NULL so checks at end of function are accurate in case XMALLOC fails.

Reply:
Not required. Both ptaA and ptaB have XMALLOC called, so any failure will set to NULL correctly.